framework/Util/TPluginModule.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+/**
+ * TPluginModule class file.
+ *
+ * @author Brad Anderson <belisoful@icloud.com>
+ * @link https://github.com/pradosoft/prado
+ * @license https://github.com/pradosoft/prado/blob/master/LICENSE
+ * @package Prado
+ */
+	
+namespace Prado\Util;
+
+use Prado\Exceptions\TException;
+use Prado\TPropertyValue;
+use Prado\Util\Behaviors\TPageServiceExtraPathsBehavior;
+use ReflectionClass;
+
+/**
+ * TPluginModule class.
+ *
+ * TPluginModule is for extending PRADO through Composer packages.  This installs
+ * its own Pages, where available, and its own error message file.
+ *
+ * Plugin pages should implement their *.page with the following page code:
+ * <code>
+ *	<com:TContent ID=<%= Prado::getApplication()->getParameters()->itemAt('PluginContentId') %>>
```

Comment:
You can probably use a property tag here:
```html
<com:TContent ID=<%$ PluginContentId %>>
```

Reply:
For some strange reason, the property tag did not work.  I'll have another look at this.

Reply:
oh right.  when using a property tag rather than = expression tag this is the error:
Invalid template syntax: TContent.ID can only accept a static text string

I don't know what the intended behavior is of the template and parsing, but it works as is without the property tag.

Reply:
given your experience, can you clarify?  I'm looking into the template parsing to see what is going on.

Reply:
https://github.com/belisoful/prado/blob/3f3ed1b1ba854d4634f6cf311bc427a481ade33e/framework/Web/UI/TTemplate.php#L846

This is why.  Adding an ID/SkinId to be keyed from the parameter needs to be coded into the template.  I have a fix. coming.